### ISSUE TITLE
[Usability] User can't identify an element or its state.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80609]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User can't identify an element or its state.
**VA.gov Experience Standard - category:** Usability
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When the "Ask for your Claim Decision" box appears on the /files page, the "View details" element is a link styled to look like a button.

With very few exceptions, we try to maintain material honesty around links and buttons. Example of why it matters: For voice command software users, if their software doesn't successfully match "Click View details" to that element then the likely next thing a user would try is "Click button." The expected behavior would be for all of the buttons on the screen to be highlighted, with prompts for the user to select the button they want. But since it's not actually a button, the View details element wouldn't be highlighted. The user then has to guess what the correct element type is, or try some other technique to get to it.

## Recommended action

Best option here is for this link to look like a link. Looks like DK has already flagged it in #78897.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jacobworrell** [comment]
this will be closed when we release the 5103 work which addresses this and other collab cycle feedback

**Comment by pmclaren19** [comment]
Follow up, this will be closed when we release the 5103 work which addresses this and other collab cycle feedback. Release expected 11/12/24.
